src/i_joystick.c

```diff
@@ -41,8 +41,12 @@ static SDL_Joystick *joystick = NULL;
 // Configuration variables:
 
 // Standard default.cfg Joystick enable/disable
-
+// always enable on Switch
+#ifdef __SWITCH__
+static int usejoystick = 1;
```

Comment:
The cleaner way to do this is to set an override in `I_BindJoystickVariables()`, eg.
```
#ifdef __SWITCH__
    usejoystick = 1;
#endif
```
Or alternatively (probably better) use `SDL_GetPlatform()` instead.